pkg/infra/process.go

```diff
@@ -4,11 +4,12 @@ import (
 	"fmt"
 	"time"
 
-	"github.com/pkg/errors"
 	log "github.com/sirupsen/logrus"
 )
 
-func Process(configPath string, num int, logger *log.Logger) error {
+const size = 1000
```

Comment:
i don't think we should make `size` un-configurable, and fixed to a rather unreasonable number (isn't 1000 too large?).

Why don't expose to user as well?

Reply:
ok, export to user as well, and default as 1000.
I don't want to export too many args to user.
Comparing previous just 10, 1000 is large enough for default.